src/diffpy/labpdfproc/mud_calculator.py

```diff
@@ -5,78 +5,78 @@
 from diffpy.utils.parsers.loaddata import loadData
 
 
-def _top_hat(x, half_slit_width):
+def _top_hat(z, half_slit_width):
     """
-    create a top-hat function, return 1.0 for values within the specified slit width and 0 otherwise
+    Create a top-hat function, return 1.0 for values within the specified slit width and 0 otherwise
     """
-    return np.where((x >= -half_slit_width) & (x <= half_slit_width), 1.0, 0)
+    return np.where((z >= -half_slit_width) & (z <= half_slit_width), 1.0, 0.0)
 
 
-def _model_function(x, diameter, x0, I0, mud, slope):
+def _model_function(z, diameter, z0, I0, mud, slope):
     """
-    compute the model function with the following steps:
-    1. Recenter x to h by subtracting x0 (so that the circle is centered at 0 and it is easier to compute length l)
+    Compute the model function with the following steps:
+    1. Recenter z to x by subtracting z0 (so that the circle is centered at 0 and it is easier to compute length l)
```

Comment:
I don't think we are using `x` correctly here either, right?  In the paper, `z` is the vertical direction centered on the center of the beam.  `x` is the direction of the x-ray beam.  We didn't define an x0 (and don't need to) but it could be something like the center of the sample.  So I am not sure that it makes sense to say "recentering z to x..."

Reply:
Yes I will change x to dz and change this docstring here